.github/workflows/build+test.yml

```diff
@@ -35,3 +35,18 @@ jobs:
         with:
           name: coverage-report
           path: "${{github.workspace}}/app/build/reports/jacoco/unitTestCoverageReport/html/"
+      - name: Store generated release build APK for stagenet
+        uses: actions/upload-artifact@v3
+        with:
+          name: apk-stagenet-release
+          path: "${{github.workspace}}/app/build/outputs/apk/staging/release/"
+      - name: Store generated release build APK for testnet
+        uses: actions/upload-artifact@v3
+        with:
+          name: apk-testnet-release
+          path: "${{github.workspace}}/app/build/outputs/apk/prodTestNet/release/"
+      - name: Store generated release build APK for mainnet
+        uses: actions/upload-artifact@v3
+        with:
+          name: apk-mainnet-release
+          path: "${{github.workspace}}/app/build/outputs/apk/prodMainNet/release/"
```

Comment:
The naming is a bit misleading. We are not building APK, we are building app bundles (AAB).
Same in the paths.

Reply:
I don't understand. It's `.apk` files in subfolders of `apk`. Do you mean that it's the wrong files?

Reply:
Yes, we are uploading .aab files to the PlayStore. The output from the builds should be .aab. With .apk every device gets all of the code and resources. 
With app bundles Google generates and serves a smaller .apk from the bundle with only the required code and resources for that particular device.

Reply:
It seems like you need some additional configuration for creating `.aab` files using Gradle, so will look into it after merging this.

But I also think we should also reconsider if it's a good idea to upload the builds for every PR update instead of only after we merge to main.

Reply:
Yes, even uploading on every merge is a bad idea. The users and testers could be overwhelmed with updates. 
For example two developers merge 4 times in a week, thats 4 updates for the app inside a week. Thats a very bad user experience.

Reply:
The uploads only stay with the workflow, so it's not going to overwhelm anyone. But it may be redundant; will file followup task to figure it out.